man/systemd.netdev.xml

```diff
@@ -1558,6 +1558,13 @@
         <filename>/dev/net/tun</filename> device.</para>
         </listitem>
       </varlistentry>
+      <varlistentry>
+        <term><varname>KeepFileDescriptor=</varname></term>
+        <listitem>
+          <para>Takes a boolean. If enabled, the file descriptor of the interface is kept. This may
```

Comment:
"kept open"

"in running state, for example while the backing process is temporarily shutdown." or something like this